.github/workflows/reusable-build-docsite.yaml

```
@@ -0,0 +1,30 @@
+name: Build the Ansible docsite
+
+on:
+  workflow_call:
+
+env:
+  PIP_DISABLE_PIP_VERSION_CHECK: 1
+
+jobs:
+  build-docsite:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repository
+        uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: "3.10"
+      - name: Install requirements
+        run: python -m pip install -r requirements.in -c requirements.txt
+      - name: Generate assets for the docs landing page
+        run: python -I -Werror build.py
+      - name: Copy sitemaps, redirects, and robots.txt files
+        run: cp -v *-sitemap.xml .htaccess robots.txt output/
+      - name: Create a downloadable archive of docs build
+        uses: actions/upload-artifact@v3
+        with:
+          name: docsite-html-assets
+          path: output/*
```

Should this be
```suggestion
          path: output/
```
? Or does that make an unnecessary subdir in the artifact?

---

Yeah it makes an unnecessary subdir.

---

Should it be `**`, then? I'm not sure if `*` matches subdirs properly.
```suggestion
          path: output/**
```

---

Yeah, fair point. Thx.